### PR TITLE
Deployment

### PR DESCRIPTION
**Pull request by craigs40**
Deploying app to Heroku

### CODE REVIEW

**Comment by craigs40** [comment]
I am getting the following error when trying to deploy this app to heroku. Any assistance regarding this issue is much appreciated.

Capture

**Review by BrianSammit — COMMENTED** [review]
CHANGES REQUIRED 🔴 🛑 
------------
Hello @craigs40, good job working on this last milestone 👏 👍 
TO HIGHLIGHT:

- Your use the correct git-flow to deploy 
- Your repo is professional 
- Your project is professional

However, there is one change to make to complete this milestone 🛠️ see it below 👇 👁️ 
-----------

- [x] Please add the link to your live demo on your Readme file. 

heroku
-----------
Please make this minor change and send it again for another review 💯 
Reach me on SLACK 
@ Brian Sammit Cruz Rodriguez
HAPPY CODING!!!

programming-happy-fast-gif

**Comment by craigs40** [comment]
I've run the following command, which revealed an ArgumentError in the environment.rb, however, I am unsure how to fix this.
rakeCapture
rakeCapture2

**Review by BrianSammit — COMMENTED** [review]
CHANGES REQUIRED 🔴 
-------
Hello there again 👋 I can see that you are working really hard on this milestone. Congrats!!!
------
- [x] Try running this command instead  ' RAILS_ENV=production bundle exec rake assets:precompile'  Make sure to use the production environment so that the production version of your assets are generated.
---------------
Please make sure your app is running locally before you send it again for another review. 
You could still write messages to get help but don't send it for review. 
Reach me on SLACK if you need help. 
HAPPY CODING!!

programming-punching-keyboard-coding-anger-13716030304

File: app/helpers/friendships_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module FriendshipsHelper
```
Comment (carlos-ssh) [inline app/helpers/friendships_helper.rb:1]: **Kindly work on these changes:**
- Provide a Link to Heroku in README file.
- Provide a Link to the deployed app is working.
Screen Shot 2021-08-09 at 18 49 03

**Review by carlos-ssh — COMMENTED** [review]
Hello team, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bellom** [comment]
## Hi @craigs40  :clap: @

### Awesome job so far!! :+1: 

# Status: Changes Required! ♻️ 

In this milestone, You need to add a live demo link to your README file.

Screenshot from the pathwright:
image

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best**

**Comment by craigs40** [comment]
Live link has already been updated in the README file under the main branch.

savecap

https://damp-tor-01710.herokuapp.com/users/sign_in

**Review by MahmoudBakr23 — COMMENTED** [review]
Hi @craigs40 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**